pkg/webhook/policy.go

```diff
@@ -157,6 +167,12 @@ func (h *validationHandler) Handle(ctx context.Context, req admission.Request) a
 		}
 	}()
 
+	// namespace is excluded from webhook using config match
+	if h.skipExcludedNamespace(req.AdmissionRequest.Namespace) {
+		requestResponse = allowResponse
+		return admission.ValidationResponse(true, "Namespace is ignored from admission because it is specified in config namespace exclusion")
```

Comment:
"in config namespace exclusion" seems confusing... maybe:

"" Namespace is set to be ignored by Gatekeeper config"